plugins/broker-redis/src/static/redis.controller.ts

```diff
@@ -0,0 +1,4 @@
+import { Controller } from "@nestjs/common";
+
+@Controller()
+export class RedisController {}
```

Comment:
Ideally we should generate the controller with endpoints that listen to specific topics.

in order to do so, have a look to this Kafka implementation
https://github.com/amplication/plugins/blob/master/plugins/broker-kafka/src/index.ts#L284

⚠️  Keep in mind that what Kafka define as Topic, Redis defines as Channel